src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/change_point/index.ts

```diff
@@ -33,5 +33,6 @@ export const changePointCommand = {
       '… | CHANGE_POINT value ON timestamp',
       '… | CHANGE_POINT value ON timestamp AS type, pvalue',
     ],
+    license: 'platinum',
```

Comment:
I am not totally sure if the metadata object is the right place (at least in this precise context) 🤔
I was trying to avoid altering the functions that handle the license by adding an extra parameter

Reply:
@bartoval we should not hardcode it.We should alter our script in order to generate commands meta from these definitions https://github.com/elastic/elasticsearch/tree/main/docs/reference/query-languages/esql/kibana/definition/commands and then retrieve the license from them